sources/brush/src/main.rs

```diff
@@ -0,0 +1,429 @@
+/*!
+# Introduction
+
+*brush* is a sanitizing pseudo-shell. Its purpose is to parse `sh -c '<program> ...'` invocations to
+decide whether the requested program should be allowed to run. It splits its third argument based
+on POSIX shell rules. Any additional input validation is left up to the invoked program.
+
+Programs can be allowed by creating a symlink in `allowed-programs` directory:
+```
```

Comment:
nit: can you force `bash` or `text` here? Otherwise the generated README uses `rust`.